### ISSUE TITLE
Missing signum congruentiae in agnostic symbols

### ISSUE THREAD

**Issue by martha-thomae** [HISPAMUS/muret#228]
I think the signum congruentiae disappeared from the agnostic symbols

![image](https://user-images.githubusercontent.com/13948831/125539470-0c56385c-c81b-49f0-8e09-f83d4f24704f.png)

**Comment by davidrizo** [comment]
It has been added again. We don't have a code in **mens to encode it yet

**Comment by martha-thomae** [comment]
True, and I think there is no MEI encoding for it as well, is it?
Thank you for adding it back into the agnostic symbols.

**Comment by davidrizo** [comment]
@martha-thomae proposes to use as a temporary solution for it to be replaced.
In **mens we are encoding it with *sc.

The correct way of encoding it right now would be by using the [``dir`` element](https://music-encoding.org/guidelines/dev/elements/dir.html). [See this MEI issue](https://github.com/music-encoding/music-encoding/issues/863).  

However, in Verovio, this element must be child of the ``measure`` element, and Verovio skips it. 
As a shortcut, we will encode it by adding a "huge" measure as the parent of all contents to be able to use the ``dir`` attribute.

**Comment by davidrizo** [comment]
We've just added [this issue](https://github.com/rism-digital/verovio/issues/2434) for Verovio to include it without measures
